modules/ocf_desktop/files/xsession/print-notify-listener

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -eu
+function clean_up {
+    jobs -pr | xargs pgrep -P | xargs sudo -u ocfbroker kill
+    exit 0
+}
+
+trap clean_up INT TERM EXIT
+
+pipe=$HOME/tmp/print-notify
+icon=/opt/share/xsession/images/ocf-color-256.png
+
+if [[ ! -p $pipe ]]; then
+    mkfifo $pipe
```

Comment:
Perhaps delete an old pipe if it already exists (for example, if a session didn't get cleaned up for some reason).